subprojects/gst-plugins-bad/sys/msdk/gstmsdkh265enc.c

```diff
@@ -101,7 +101,7 @@ enum
 #define PROP_INTRA_REFRESH_CYCLE_DIST_DEFAULT 0
 #define PROP_DBLK_IDC_DEFAULT                 0
 
-#define RAW_FORMATS "NV12, I420, YV12, YUY2, UYVY, BGRA, BGR10A2_LE, P010_10LE, VUYA"
```

Comment:
According to media-driver https://github.com/intel/media-driver#decodingencoding-features, 8bit/422 is supported for hevc encoder from platforms > ICL, so we may include yuy2 in the caps and add a platform check somewhere else (like in gst_msdkenc_init_encoder ? ).

Reply:
Thank you, @mengker33 , I will make further confirmation of all the formats.